website/content/docs/concepts/security/permissions/grants.mdx

```diff
@@ -0,0 +1,186 @@
+
+**Actions** convey the ability to perform some action against a resource or collection.
+You can select which actions you want to apply to a grant.
+You can also use a [wildcard](/boundary/docs/concepts/security/permissions/permission-grant-formats#wildcard-id) to include all valid actions.
+Each grant string can include one or more comma separated actions.
+Some commonly used actions include:
+
+- &#32 `*` (all)
+- &#32 `list`
+- &#32 `create`
+- &#32 `read`
+- &#32 `update`
+- &#32 `delete`
+
+To learn more about actions, see [advanced actions](/docs/concepts/security/permissions/advanced#advanced-actions).
+For a full list of actions, see the [Resource Table](/docs/concepts/security/permissions/resource-table).
+
+## Output Fields
+
+**Output fields** are top-level fields that are returned in the response. Output fields
+are **optional** and not commonly used.
+
+Examples of valid output fields include:
+
+- &#32 `none`
+- &#32 `id`
+- &#32 `name`
+- &#32 `description`
+
+To learn more about output fields, see [output fields](/docs/concepts/security/permissions/advanced#output-fields)
+
+## Common Grant Examples
+
+The following are examples of commonly used grant strings for specific roles and scopes.
+
+**All Actions** 
+
+This grant string allows users to perform any actions on any resources in the scope:
+
+- &#32 `id=*;type=*;actions=*`
+
+**View All** 
+
+This grant string allows users to list and read any resources in the scope:
+
+- &#32 `id=*;type=*;actions=read,list`
+
+**View Targets Only** 
+
+This **project scope** grant string allows users to list and read any targets in the specific project.
+This example is only applicable to project scopes because targets can only exist inside a project:
+
+- &#32 `id=*;type=target;actions=read,list`
+
+**Connect To Any Target**
+
+This **project scope** grant string allows users to list, read, and connect to any targets in a specific project.
+In addition, these grants allow the user to list, read, and cancel any sessions initiated by the same user.
+This example is only applicable to project scopes, since targets can only exist in a project:
+
+- &#32 `id=*;type=target;actions=list,read,authorize-session`
+- &#32 `id=*;type=session;actions=read:self,cancel:self,list`
+
+**Connect To A Specific Target** 
+
+This **project scope** grant string allows users to list, read, and connect to a specific target.
+In addition, these grants let users list, read, and cancel any sessions initiated by the same user.
+If you want to include several specific targets, you can duplicate the first grant string using the respective IDs of those targets.
+This example is only applicable to project scopes, since targets can only exist in a project:
+
+- &#32 `id=<target-id>;actions=read,authorize-session`
+- &#32 `type=target;actions=list`
+- &#32 `id=*;type=session;actions=read:self,cancel:self,list`
+
+<Note>
+There is a known issue that affects the [Project] Connect To A Specific Target grant strings.
```

Comment:
Isn't this known issue fixed in 0.13.0? Maybe we can remove this.

Reply:
I need to validate this first, will do soon